docs/maps/connect-to-ems.asciidoc

```diff
@@ -0,0 +1,28 @@
+[role="xpack"]
+[[maps-connect-to-ems]]
+== Connecting to Elastic Maps Service
+
+https://www.elastic.co/elastic-maps-service[Elastic Maps Service (EMS)] is a service that hosts
+tile layers and vector shapes of administrative boundaries.
+With Kibana's out-of-the-box settings, the **Maps** application uses EMS services.
+
+Requests to EMS are made with two methods:
+
+* Requests for tile layer meta data, vector shapes meta data, and vector shapes are proxied through the Kibana server.
+* Requests for EMS tiles are made directly from the browser to EMS.
+
+NOTE: The legacy tilemap and regionmap visualizations do not proxy any EMS requests through the Kibana server.
+Requests are made directly from the browser to EMS.
+
+
+[float]
+=== Disabling Elastic Maps Service
+
+You might experience EMS connection issues if your Kibana server or browser are on a private network or
+behind a firewall. If this happens, you can disable the EMS connection to avoid unnecessary EMS requests.
+
+To disable EMS, change your <<settings, kibana.yml>> file.
+
+. Set `map.includeElasticMapsService: false` to turn off the EMS connection.
```

Comment:
Let's make this list parallel:

Set `map.includeElasticMapsService` to `false` to turn off the EMS connection.